modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_compare/MiniCompare.js

```diff
@@ -197,9 +184,15 @@ function MiniCompare(props) {
 					);
 				})}
 
-			<a className="btn btn-primary" href={props.compareProductsURL}>
-				{Liferay.Language.get('compare')}
-			</a>
+			{renderButton}
```

Comment:
Could you render button directly here instead?

Reply:
Fixed